app/src/app/views/admin-page/default-marking-definitions/default-marking-definitions.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="view-page grid spacing-16">
+    <div class="row">
+        <div class="col">
+            <h1>Default Marking Definition Statements</h1>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col">
+            <p>
+                Default Marking Definition Statements are statements that will be applied by default to newly created objects.
+                The default marking definition statements will also be applied to objected that were saved and did not have the default statements. 
+            </p>
+        </div>
+    </div>
+    <div class="grid">
+        <hr>
+        <div class="row">
+            <div class="col center">
+                <div>
+                    <button mat-mini-fab class="extended-fab-button" color="primary" (click)="updateDefaultMarkingDefs()">
+                        <mat-icon>settings</mat-icon> <span class="text-label">update default marking definitions</span>
+                    </button>
+                </div>
+            </div>
```

Comment:
I like the approach you've taken, but there may be a simpler solution. Rather than a popover, what if we displayed a STIX list of the currently selected default marking definitions underneath the button? If we added `clickBehavior: none` to the STIX list config, users wouldn't be able to interact with the table itself and could easily check the current selection to determine whether or not it needs to be updated